roles/mesos/tasks/main.yml

```diff
@@ -23,6 +23,10 @@
   include: RedHat.yml
   when: ansible_os_family == "RedHat"
 
+- name: remove old NS
+  lineinfile: dest={{mesos_dns_config_file[ansible_os_family]}} regexp="^nameserver.*" line="nameserver {{item}}" state=absent
```

Comment:
I will review other places where we use `lineinfile` to ensure we support modifications.
BTW, to avoid changes it might make sense to manage DNS config files via templates.